extras/src/runtimes.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// substrate-desub is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// substrate-desub is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with substrate-desub.  If not, see <http://www.gnu.org/licenses/>.
+
+use phf::phf_ordered_map;
+
+pub fn kusama_upgrade_block(version: &u64) -> Option<u64> {
+	KUSAMA_RUNTIMES.get(version).copied()
+}
+
+static KUSAMA_RUNTIMES: phf::OrderedMap<u64, u64> = phf_ordered_map! {
+	0u64       => 1020,
+	26669u64   => 1021,
+	38245u64   => 1022,
```

Comment:
The `block_number => version` should be swapped around since we are getting by version (alas both are u64 so Rust didn't catch it!) :)